### PR TITLE
HPCC-13229 Improve error reporting of #WEBSERVICE and other #commands

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday gavin.halliday@lexisnexis.com

### CODE REVIEW

**Comment by ghalliday** [comment]
@richardkchapman @jamienoss please review.

**Comment by jamienoss** [comment]
@ghalliday this all looks good. How did the regression comparison go?
I think we should have a regression test for this.

**Comment by ghalliday** [comment]
There are regression tests for it already - errorwebservice.ecl and some others.

**Comment by ghalliday** [comment]
@richardkchapman

File: ecl/hql/hqlgram.y [hunk 1]
```diff
@@ -2274,9 +2274,11 @@ action
                         {
                             $1.annotateExprWithLocation();
                             $$.inherit($1);
-                            $$.setPosition($1);
```
Comment (richardkchapman) [inline ecl/hql/hqlgram.y]: Was this line removed deliberately?

Comment (ghalliday) [inline ecl/hql/hqlgram.y]: Yes, the $$.inherit will already have done that.

**Pull request merged**